aggregating-Apache-logs-with-fluentd-and-Minio-server.md

```diff
@@ -43,12 +48,12 @@ We are replacing `/etc/td-agent/td-agent.conf` with:
    buffer_chunk_limit 256m
 </match>
 ```
-After this please restart your `fluentd` server.  
+Step 3 : Restart `fluentd` server.  
 ```
```

Comment:
sure.

On Mon, Jul 4, 2016 at 12:42 AM, GarimaKapoor <notifications@github.com>
wrote:

> In aggregating-Apache-logs-with-fluentd-and-Minio-server.md
> <https://github.com/minio/howto/pull/11#discussion_r69396063>:
>
> > @@ -43,12 +48,12 @@ We are replacing `/etc/td-agent/td-agent.conf` with:
> >     buffer_chunk_limit 256m
> >  </match>
> >  ```
> > -After this please restart your `fluentd` server.
> > +Step 3 : Restart `fluentd` server.
> >  ```
>
> Should be Step3:
> No space between 3 and :
>
> Similarly correct other steps aswell.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/minio/howto/pull/11/files/2f19f7b2d77413f5cc95240c5eca59e5a1b1842f#r69396063>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AACjEpIhpbAPt1yzCLrHG7_1xMrTFV3Vks5qSAmZgaJpZM4JC7JL>
> .
>



-- 
Atul Jha
Developer Evangelist
Web: www.minio.io
Github: https://github.com/minio
Chat: https://www.gitter.im/minio/minio
YouTube: https://www.youtube.com/channel/UCnWYd9f4il3tyshx5M9lrGw
<https://www.youtube.com/channel/UCnWYd9f4il3tyshx5M9lrGw>